travis/build_info.py

```diff
@@ -0,0 +1,207 @@
+"""
+Gather Travis usage info in real-time using Travis REST APIs
+
+This only applies to travis instances that do not require
+authorization (e.g., travis-ci.org). Auth is a #TODO
+
+"""
+
+import argparse
+import logging
+import requests
+import sys
+
+
+logger = logging.getLogger(__name__)
+
+BASE_URL = 'https://api.travis-ci.org/'
+
+
+def get_repos(org):
+    """
+    Returns list of active repos in a given org.
+    """
+
```

Comment:
I realize it's a style thing, so whatever you decide is totally cool. I'd suggest though that we not have a blank line under docstrings. IMO it's not needed for readability and makes the files longer.